pkg/machine/shim/networking.go

```diff
@@ -73,11 +73,9 @@ func startNetworking(mc *vmconfigs.MachineConfig, provider vmconfigs.VMProvider)
 	// GvProxy PID file path is now derived
 	cmd.PidFile = filepath.Join(runDir.GetPath(), "gvproxy.pid")
 
-	// TODO This can be re-enabled when gvisor-tap-vsock #305 is merged
-	// debug is set, we dump to a logfile as well
-	// if logrus.IsLevelEnabled(logrus.DebugLevel) {
-	// 	cmd.LogFile = filepath.Join(runDir.GetPath(), "gvproxy.log")
-	// }
+	if logrus.IsLevelEnabled(logrus.DebugLevel) {
+		cmd.LogFile = filepath.Join(runDir.GetPath(), "gvproxy.log")
+	}
```

Comment:
Can you please add a log line here saying something like `gvproxy log file located at $path` so it can be discovered.

Reply:
The gvproxy command-line that was run will be logged which will contain the log file path [1]. But I can add this separately if you think that will be better.

[1]
```bash
DEBU[0000] gvproxy command-line: /usr/local/opt/podman/libexec/podman/gvproxy -debug -mtu 1500 -ssh-port 50693 -listen-vfkit unixgram:///var/folders/rt/0hzb6x897sq5v50c0p3xyggc0000gn/T/podman/podman-machine-default-gvproxy.sock -forward-sock /Users/jakecorrenti/.local/share/containers/podman/machine/applehv/podman.sock -forward-dest /run/user/501/podman/podman.sock -forward-user core -forward-identity /Users/jakecorrenti/.local/share/containers/podman/machine/machine -pid-file /var/folders/rt/0hzb6x897sq5v50c0p3xyggc0000gn/T/podman/gvproxy.pid -log-file /var/folders/rt/0hzb6x897sq5v50c0p3xyggc0000gn/T/podman/gvproxy.log
```

Reply:
Ah good, that works for me as well. I just wanted to see the final path somewhere.